docs/overview/obj.rst

```diff
@@ -252,7 +252,7 @@ display using :cpp:expr:`lv_disp_set_default(disp)`.
 :cpp:func:`lv_screen_active`, :cpp:func:`lv_screen_load` and :cpp:func:`lv_screen_load_anim` operate
 on the default display.
 
-Visit `Multi-display support <disp>` to learn more.
+Visit :ref:`Multi-display support <display_multi_display_support>` to learn more.
```

Comment:
This is wrong. It should read 

:ref:`display_multi_display_support`

The title of the section will automatically get used as the link text.

Reply:
Well, I thought to keep some labels and only reference other page.
Like the syntax mentioned here:
https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#explicit-targets

Reply:
There is no need to specify the link text. It will pull the link text in from the heading on the page the reference is defined above. It's just extra keystrokes that you don't need to do unless for some reason you which to change the link text that is being displayed. It's up to up. It's more work is all. 


doing this 

:ref:\`Multi-display support \<display_multi_display_support\>\`

will produce a  the link text

```
Multi-display support
```

and using this

:ref:\`display_multi_display_support\`

will produce the link text
```
Multiple display support
```

Sphinx when using the references will go and see where the reference is which is this

"""
.. _display_multi_display_support:

Multiple display support
************************
"""

and it grabs that text you see right below the reference definition to sue as the text for the link when :ref:\`display_multi_display_support\` is used